homeassistant/components/sensor/serial.py

```diff
@@ -91,6 +113,11 @@ def should_poll(self):
         return False
 
     @property
+    def state_attributes(self):
```

Comment:
Don't overwrite `state_attributes`, use `device_state_attributes`.

Reply:
Can't you elaborate on that ? Or point me towards some example/documentation on the differences?

I basically followed the REST sensor example, but I'm available to improve the code.

Reply:
The former should only be implemented by base components, the latter by platforms.

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py#L104-L117

Reply:
I get it, but it's currently just semantics. The pull request is closed, can any of view just make the change :) ?

Reply:
Just make a new PR with the change and we can quickly merge it.